### PR TITLE
Added prop-in-wall save

### PR DESCRIPTION
**Pull request by Ramalayha**
Automatically saves when you push a prop into a wall.

### CODE REVIEW

File: spt/OrangeBox/modules/ClientDLL.cpp [hunk 1]
```diff
@@ -415,6 +421,43 @@ void ClientDLL::ResetAfterframesQueue()
...
+	if (y_spt_piwsave.GetString()[0] != '\0')
+	{
+		auto ply = GetEntity(1);
+		if (ply)
+		{
+			auto pphys = ply->VPhysicsGetObject();
+			if (pphys && (pphys->GetGameFlags() & FVPHYSICS_PENETRATING) == 0)
+			{
+				int count = GetEntityCount();
+				for (int i = 0; i < count; ++i)
```
Comment (mikesmiffy128) [inline spt/OrangeBox/modules/ClientDLL.cpp]: I might well be wrong here, but if worldspawn is 0 and the player is 1, isn't it unnecessary to start this loop at index 0, and you could use 2 instead? Not that it'd make a big difference anyway.

**Comment by YaLTeR** [comment]
Alright so change of plans: since SPT's code base contains a bunch of stuff without checks that works only on one game version, this can go through as long as it doesn't crash the game when the cvar is switched off. Could you confirm that's the case? (and resolve the conflict?)

**Comment by Margen67** [comment]
@Ramalayha :^)

**Comment by YaLTeR** [comment]
Jukspa updated this over in #136

**Pull request closed without merging**